src/frameworks/vite/index.ts

```diff
@@ -105,7 +106,8 @@ export async function getDevModeHandle(dir: string) {
     const serve = spawn(cli, [], { cwd: dir });
     serve.stdout.on("data", (data: any) => {
       process.stdout.write(data);
-      const match = data.toString().match(/(http:\/\/.+:\d+)/);
+      const dataWithoutAnsiCodes = stripAnsi(data.toString());
+      const match = dataWithoutAnsiCodes.match(/(http:\/\/.+:\d+)/);
```

Comment:
nit: we could strip ansi only if there's a match to avoid doing it unnecessarily?